src/components/home/modals/CreateToken.vue

```diff
@@ -713,8 +713,8 @@ export default {
             if (this.selectedNFTContract) {
                 let lastTokenIndexRes = await this.chain.getLastTokenIndex(this.nftContractID)
                 let newTokenIndex = 0
-                if (lastTokenIndexRes.hasOwnProperty('lastTokenIndex')) {
-                    newTokenIndex = lastTokenIndexRes['lastTokenIndex']
+                if (lastTokenIndexRes.hasOwnProperty('lastTokenIndex') || (lastTokenIndexRes.hasOwnProperty('error') && lastTokenIndexRes['error'] === 199)) {
+                    newTokenIndex = lastTokenIndexRes['lastTokenIndex'] ? lastTokenIndexRes['lastTokenIndex'] + 1 : newTokenIndex
```

Comment:
There is bug for issuing the 2nd NFT. For the last Token index API response return 0, the `lastTokenIndexRes['lastTokenIndex']` will be 0, which means false in this condition. The Token ID will be generated as the 1st NFT. Please change `lastTokenIndexRes['lastTokenIndex']` to `lastTokenIndexRes.hasOwnProperty('lastTokenIndex')`.